app_feup/.gitignore

```diff
@@ -71,6 +72,10 @@ build/
 **/ios/Flutter/app.flx
 **/ios/Flutter/app.zip
 **/ios/Flutter/flutter_assets/
+<<<<<<< HEAD
+**/ios/Flutter/flutter_export_environment.sh
+=======
+>>>>>>> develop
```

Comment:
WTF is this???

Reply:
Somehow it let me commit merging files? This tends not to happen but thank you for the heads up.